libraries/rib-compose/src/main/java/com/badoo/ribs/compose/ComposeRibView.kt

```diff
@@ -50,6 +53,14 @@ abstract class ComposeRibView(
 
     override fun detachChild(child: Node<*>, subtreeOf: Node<*>) {
         child.onDetachFromView()
-        getParentViewForSubtree(subtreeOf).value = null
+
+        // Only detach the same child, or we would remove something unintended.
+        // If there was already another child attached, then this one in
+        // MutableState was overwritten, and already removed from the composition as a result,
+        // so no further action is needed.
+        if (child == lastChildAttached) {
```

Comment:
With the current implementation we have an important difference between `AndroidView` and `ComposeView` attach logic.
For `AndroidView` we **add** the child to the view group. So other children are still there when new one is attached.
But for `ComposeView` we are **replacing**  the previous child in `target:mutableState`. So the default configuration essentially support only one child to be shown. Which is actually the reason of the bug we're fixing in this PR.

Is this difference in attach behavior intended? Users may expect `ComposeView` to behave like `AndroidView`, maybe we should make them the same, or document the differences.